index.d.ts

```diff
@@ -97,6 +98,7 @@ interface InitConfig {
     messagingSenderId?: string
     // "G-MEASUREMENT_ID"
     measurementId?: string
+    tenantId?: string
```

Comment:
Let's move this out of `firebaseClientInitConfig` and onto the top level. For ease of use, I think it's useful to leave `firebaseClientInitConfig` as purely the inputs to `initializeApp`.

Reply:
This has now been fixed